.circleci/config.yml

```diff
@@ -1,9 +1,9 @@
-version: 2
+version: 2.1
 jobs:
   build:
     working_directory: ~/digdag-slack
     docker:
-      - image: openjdk:8u141-jdk
+      - image: cimg/openjdk:8.0.402
```

Comment:
openjdk:8u141-jdk was added Let's Encrypt certificates ( https://www.java.com/download/help/release_changes.html ).
However, that CA had already expired, so an error occurs as below ( https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/)
```
curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
```
I modified to use the latest cimg/open-jdk8 instead of openjdk:8u141-jdk.